cli/dcoscli/package/main.py

```diff
@@ -160,9 +160,10 @@ def _update(validate):
     :rtype: int
     """
 
-    config = util.get_config()
+    emitter.publish("This command is being deprecated")
```

Comment:
Hmm. I think we should only print this when the CLI is configure to talk to `Cosmos`. Eventually, we are going to add code that auto discover if Cosmos is installed and configure it automatically.